artist/src/main/java/com/ataulm/artcollector/artist/ui/ArtistViewModel.kt

```diff
@@ -15,11 +18,15 @@ import kotlinx.coroutines.withContext
 import javax.inject.Inject
 
 internal class ArtistViewModel @Inject constructor(
+        private val getArtist: GetArtistUseCase,
         private val getArtistGallery: GetArtistGalleryUseCase
 ) : ViewModel() {
 
+    private val _artist = MutableLiveData<Artist>()
     private val _gallery = MutableLiveData<Gallery>()
-    val gallery: LiveData<Gallery> = _gallery
```

Comment:
we don't want to expose `gallery` anymore, we'll use the custom Mediator below so we can expose `ArtistGallery` instead